src/SignhostAPIClient/Rest/ISignhostApiReceiver.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Signhost.APIClient.Rest;
+using Signhost.APIClient.Rest.DataObjects;
+
+namespace Signhost.APIClient.Rest
+{
+	public interface ISignhostApiReceiver
+	{
+		bool IsPostbackChecksumValid(IDictionary<string, string[]> headers, string body, out Transaction postbackTransaction);
```

Comment:
Please add a documentation to both the interface and the method including it's parameters so callee's now what to expect / provide.